source/documentation/other-topics/rds-snapshots.html.md.erb

```diff
@@ -15,3 +15,59 @@ Examples about managing RDS snapshots using AWS CLI can be found in the [README]
 - The amount of manual snapshot is limited per account - so it's important to cleanup
 - Daily snapshots are already happening out of the box. It's important to keep in mind that daily snapshots **are not related** with "Manual Snapshots"
 - Managing snapshots are the team responsibilities (including a snapshot restores). Teams are also responsible for cleaning up any manual snapshots in order to avoid hitting our AWS account limits
+
+## Restoring live services from a RDS snapshot
+
+If you want to restore your production RDS instance from a previous snapshot either because the database is corrupted or the whole database is deleted, here is the procedure to restore it from your RDS snapshot.
+
+1. Get the `DBSnapshotIdentifier` of your DB instance from which you want to restore using AWS CLI below:
+   
+    ```
+    aws rds describe-db-snapshots --db-instance-identifier [db-instance-identifier]
+    ```
+    The `db-instance-identifier` is the first part (`cloud-platform-xxxxx`) of `rds_instance_address` in your kubernetes secret inside your namespace. 
```

Comment:
It would be really nice to have something that looked like a real rds_instance_address value as an example here, so that when you say "first part", the reader understands what that means. So, something like;

e.g. if your rds_instance_address value is; cloud-platform-foo/bar/baz|whatever:something`, then the `db-instance-identifier` is `cloud-platform-foo`